PlayStore_donation/fr-FR.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?><!--
+text.xml file from Click to chat app (donation version) by TrianguloY.
+Permission given to translate only.
+Instructions for manual translation:
+- Translate the properties making sure not to overpass the maximum character number.
+- Currently this file is copied manually, line breaks are kept and no need to escape special characters.
+If you added/modified the following file add your name here to give credit:
+
+    <!--<string name="translators">Nathanaël Gagnepain</string>-->
+ -->
+<parameters>
+
+    <!-- Preferably use the name from https://faq.whatsapp.com/general/chats/how-to-use-click-to-chat/ in your language -->
+    <title max_chars="30">Cliquer pour discuter [donateur]</title>
```

Comment:
This title is 32 chars, and Play Store only allows 30 (that's so close!). Any synonym with fewer letters? Maybe '[don]'? (according to google translate)

Reply:
Also done, sorry for these mistakes